lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Action.js

```diff
@@ -15,11 +15,13 @@ qx.Class.define("callbackery.ui.plugin.Action", {
      *
      * @param vizWidget {Widget} visualization widget to embedd
      */
-    construct : function(cfg,buttonClass,layout,getFormData) {
+    construct : function(cfg,buttonClass,layout,getFormData, plugin) {
```

Comment:
hmmm the documentation seems to be out of sync

Reply:
Nicht nur hier und nicht nur seit diesem PR ... hab's in alle hier geänderten Files ergänzt